pretender.js

```diff
@@ -15,8 +15,7 @@ var FakeXMLHttpRequest = appearsBrowserified ? getModuleDefault(require('fake-xm
   self.FakeXMLHttpRequest;
 
 // fetch related ponyfills
-// TODO: use whatwg-fetch once new version release
-var FakeFetch = appearsBrowserified ? getModuleDefault(require('@xg-wang/whatwg-fetch')) : self.WHATWGFetch;
+var FakeFetch = appearsBrowserified ? getModuleDefault(require('whatwg-fetch')) : self.WHATWGFetch;
```

Comment:
I noted in #221 thinking that it's a documentation bug, but `self.WHATWGFetch` is undefined for me.